### DISCUSSION TITLE
2023년 07월 27일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#70] [General]
# 2023년 07월 27일

LOG 아침부터 의자에 올라가고 싶은 날이었다.. 가끔 아침부터 에너지가 주체 안 되는 날이 있는데 그게 바로 오늘이었다. 끝나고 잠실에 놀러갔다. 토리, 포이, 망고랑 저녁으로 신천직화를 먹었다. 오랜만에(?) 잠실 와서 그런가 13층인지 14층인…

https://vero.wiki/2023-07-27/

**Comment by carsago** [comment]
휴 2주 밀렸던거 몰아봤다

**Reply by Cyma-s** [reply]
미리미리 좀 보세요
